sdk/storage/Azure.Storage.Files.DataLake/tests/FileSystemClientTests.cs

```diff
@@ -465,9 +465,6 @@ public async Task DeleteIfExistsAsync_Exists()
 
             // Act
             response = await fileSystemClient.DeleteIfExistsAsync();
-
-            //Assert
-            Assert.IsFalse(response.Value);
```

Comment:
Might be worth leaving the comment why this happens. Test without assertion looks suspicious. Maybe instead of removing it we could find some other thing to assert to prove that operation was successful (unless absence of exception is the only thing we're looking for)

Reply:
We're going for just not throwing an exception on line 467.